src/app/core/components/main-header/main-header.component.ts

```diff
@@ -40,6 +40,9 @@ export class MainHeaderComponent implements OnInit, OnDestroy {
                   this.profileData = res['data'] as any;
                   this.isLoggedIn = true;
                   console.log(this.profileData);
+                  if (this.profileData.profile_picture == null) {
```

Comment:
Can we do this with a ternary operator inside the html template instead of doing it here?